tests/bench/test_powerlift.py

```diff
@@ -0,0 +1,25 @@
+import pytest
+import tempfile
+
+from guidance.bench._powerlift import retrieve_langchain
+from pathlib import Path
+
+def test_retrieve_langchain_err(monkeypatch):
```

Comment:
monkeypatch is a fixture?

Reply:
Yes, it's an in-built fixture within pytest.